faker/providers/ssn/en_US/__init__.py

```diff
@@ -4,4 +4,17 @@
 
 
 class Provider(BaseProvider):
-    pass
+
+    @classmethod
+    def ssn(cls):
+        # Certain numbers are invalid for U.S. SSNs. The area (first 3 digits)
+        # cannot be 666 or 900-999. The group number (middle digits) cannot be
+        # 00. The id (last 4 digits) cannot be 0000
+        area = BaseProvider.random_int(min=1, max=899)
+        if area == 666:
+            area += 1
+        group = BaseProvider.random_int(1, 99)
+        serial = BaseProvider.random_int(1, 9999)
+
+        ssn = "{:03d}-{:02d}-{:04d}".format(area, group, serial)
```

Comment:
We're still supporting python 2.6, which doesn't support this kind of string formatting.

You can rewrite this as `"{0:03d}-{1:02d}-{2:04d}".format(area, group, serial)` or ` "%03d-%02d-%04d" % (area, group, serial)`